### PR TITLE
caffeinate: add -m example

### LINKED ISSUE #9167: page request: caffeinate
### Command description

Add new key option - https://ss64.com/osx/caffeinate.html

### Command details

_No response_

### OS

- [ ] Android
- [ ] Linux
- [X] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by Nikhil-Samant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
closes #9167

### CODE REVIEW

File: pages/linux/caffeinate.md [hunk 1]
```diff
@@ -6,3 +6,7 @@
 - Prevent desktop from sleeping (use `Ctrl + C` to exit):
 
 `caffeinate`
+
+- Prevent disk from sleeping until you type Ctrl-C:
+
+`caffeinate -m`
```
Comment (navarroaxel) [inline pages/linux/caffeinate.md:12]: The linux command doesn't have this `-m` flag

File: pages/osx/caffeinate.md [hunk 2]
```diff
@@ -14,3 +14,7 @@
 - Prevent from sleeping until you type Ctrl-C:
```
Comment (navarroaxel) [inline pages/osx/caffeinate.md:14]:
```suggestion
- Prevent from sleeping (use `Ctrl + C` to exit):
```

File: pages/osx/caffeinate.md [hunk 3]
```diff
@@ -14,3 +14,7 @@
...
 
 `caffeinate -i`
+
+- Prevent disk from sleeping until you type Ctrl-C:
```
Comment (navarroaxel) [inline pages/osx/caffeinate.md:18]:
```suggestion
- Prevent disk from sleeping (use `Ctrl + C` to exit):
```

File: pages/linux/caffeinate.md (hunk 1 above)
Comment (Nikhil-Samant) [reply pages/linux/caffeinate.md:12]: Sure. Will have a look

Comment (Nikhil-Samant) [reply pages/linux/caffeinate.md]: I agree, on linux the command doesn't have the flag. Reverted the changes

File: pages/osx/caffeinate.md (hunk 2 above)
Comment (navarroaxel) [reply pages/osx/caffeinate.md:14]: it's not applied yet

Comment (Nikhil-Samant) [reply pages/osx/caffeinate.md]: @navarroaxel , It's done now

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**